perses/app/setup_relative_calculation.py

```diff
@@ -97,21 +131,20 @@ def getSetupOptions(filename):
 def run_setup(setup_options):
     """
     Run the setup pipeline and return the relevant setup objects based on a yaml input file.
-
     Parameters
     ----------
     setup_options : dict
         result of loading yaml input file
-
     Returns
     -------
-    fe_setup : NonequilibriumFEPSetup
-        The setup class for this calculation
-    ne_fep : NonequilibriumSwitchingFEP
-        The nonequilibrium driver class
+    setup_dict: dict
+        {'topology_proposals': top_prop, 'hybrid_topology_factories': htf, 'hybrid_samplers': hss}
+        - 'topology_proposals':
     """
-    from perses.tests.utils import validate_endstate_energies
     phases = setup_options['phases']
+    if len(phases) > 2:
```

Comment:
I thought we decided to take this out? Sometimes I want to run complex/solvent/vacuum all at once.

Maybe we just throw a warning if there's only 1 leg that you need 2 for a proper free energy?